src/main/java/study/racing/domain/Distance.java

```diff
@@ -0,0 +1,24 @@
+package study.racing.domain;
+
+public class Distance {
+
+    private static final int DEFAULT_DISTANCE = 0;
+
+    int moveDistance;
+
+    public Distance(int moveDistance) {
+        this.moveDistance = moveDistance;
+    }
+
+    public Distance() {
+        this.moveDistance = DEFAULT_DISTANCE;
+    }
+
+    public int getMove() {
```

Comment:
```suggestion
    public int getDistance() {
```
가 좀더 명확하게 의미를 전달해줄 수 있을 것 같아요.

Reply:
네 메소드명 변경하겠습니다.